www/js/Controller/My_item/Myitems.js

```diff
@@ -128,7 +128,7 @@ elephant.controller('MyitemsController', function($scope, $http, $timeout, $loca
           $scope.reApprove(dataString);
```

Comment:
What is the user intention to use "re Approve". If it's about resubmitting the item, how are we dealing with the user journey on the from My Items page to Post Item or somewhere else?